include/deal.II/base/function.h

```diff
@@ -310,6 +310,33 @@ class Function : public FunctionTime<Number>,
                                       std::vector<Vector<Number> >   &values) const;
 
   /**
+   * Compute the Hessian of a given component at point <tt>p</tt>.
```

Comment:
Can you add a definition of what the "Hessian" actually is? I don't think it is universally understood as simply the second derivative (unless you are an optimization person, of course).